src/desktop/apps/auth2/components/ModalContainer.tsx

```diff
@@ -10,14 +10,15 @@ export const ModalContainer: React.SFC<any> = () => {
   mediator.on('open:auth', options => {
     setCookies(options)
 
-    // TODO: remember to swap 'register' with 'signup' in triggers
     if (options && options.mode === 'register') {
       options.mode = 'signup'
     }
 
-    if (manager) {
-      manager.openModal(options)
-    }
+    setTimeout(() => {
+      if (manager) {
+        manager.openModal(options)
+      }
+    }, document.readyState === 'complete' ? 0 : 500)
```

Comment:
Interestingg, in what situation is the document not complete by the time we hit this? I feel like this code happens fairly later on since it's initiated from a user action but I might be missing something. In any case, it might be worth documenting inline!

Reply:
This fixes the `/log_in` and `/sign_up` use case where the modal is triggered as soon as the DOM is ready. It causes some problems with the animation which sets opacity to 0 sporadically

Reply:
Gotcha, that makes sense.